articles/machine-learning/preview/how-to-use-model-data-collection.md

```diff
@@ -34,6 +34,12 @@ Then run the following command:
 
     pip install azureml.datacollector
 
+## Set environment variables
+
+Model data collection depends on two environment variables. AML_MODEL_DC_STORAGE_ENABLED must be set to true (all lowercase) and AML_MODEL_DC_STORAGE must be set to the connection string for the Azure Storage account where you want to store the data.
+
+These environment are already set for you when the Web service is running on a cluster in Azure. When running locally you have to set them yourself. If you are using Docker, use the -e parameter of docker run command to pass environment variables.
```

Comment:
1st sentence: Add "variables" - "These environment variables..."
1st sentence: "Web" should be lowercase - "...when the web service is running..."
3rd sentence: Add "the" - "...parameter of the docker run command..."